apis/nucleus/src/viz.js

```diff
@@ -237,6 +272,28 @@ export default function viz({
       await rendered;
       return cellRef.current.takeSnapshot();
     },
+    // ===== undocumented experimental API - use at own risk ======
+    /**
+     *  valid types: viewData, cssScaling, snapshot, exportData, exploration
+     *  questionable types: supportRefresh, quickMobile, fullscreen
+     *  deprecated?: sharing
+     *
+     */
+    support(type) {
+      if (viewDataObjectId !== undefined) {
+        throw new Error('Cannot call support when in data view');
+      }
```

Comment:
This is not ok. It throws the error on any type when the chart is in view data mode. Will cause code for other things not working

Reply:
thats the idea, you are not supposed to call this when you are in viewDataMode as it will return incorrect settings. 
What would you propose instead?

Reply:
But then we have to check if we are in viewData in conjunction with the usage of `support` function every time we want to use the support function. It is kind of strange to have two functions that always have to use together. I don't have an idea yet. I will think about this. 
```
let support = false;
if (!viz.viewDataToggled) {
    support = viz.support("type");
}
```

Reply:
The problem is to know what we should return, either always the settings for the base chart or for the table? What would we expect?

Reply:
I think it should always return the base chart and not the table.

Reply:
Since you can't do any thing in context menu when you are in view data we are not gonna need to know anything related to "table extension" so I think it should alway return things for the original chart type just like it is know in sense-client.

Reply:
fair, i'll change to that